### PR TITLE
Crazy refactor compat

### PR DESCRIPTION
**Pull request by pdewouters**

### CODE REVIEW

**Comment by pdewouters** [comment]
@owaincuvelier review merge please

File: inc/class-transfer.php [hunk 1]
```diff
@@ -102,9 +105,9 @@ protected function get_connection( $region ) {
 
 			// establish our credentials
 			if ( 'LON' === $region )
-				$this->connection = new Rackspace( RACKSPACE_UK, $credentials );
+				$this->connection = new \Rackspace( RACKSPACE_UK, $credentials );
```
Comment (owaincuvelier) [inline inc/class-transfer.php]: Got this fatal error when saving credentials ```
Fatal error: Class 'Rackspace' not found in /srv/www/hmbase.dev/content/plugins/backupwordpress-pro-rackspace/inc/class-transfer.php on line 108```

**Comment by owaincuvelier** [comment]
When running a backup now, getting this fatal: ```Fatal error: Call to undefined method HM\BackUpWordPress\Scheduled_Backup::get_archive_filepath() in /srv/www/hmbase.dev/content/plugins/backupwordpress-pro-rackspace/inc/class-transfer.php on line 57
```. 

presumably a namespacing issue

**Comment by owaincuvelier** [comment]
i haz merged.

**Comment by pdewouters** [comment]
did the upload work for you then?

**Comment by owaincuvelier** [comment]
@pdewouters Yup, two confirmed files on rackspace

**Pull request merged**